### ISSUE TITLE
floatingFilter not keep value when call setRowData on version 27.0.1

### ISSUE THREAD

**Issue by nguyenhuylinhdev** [ag-grid/ag-grid#4955]
```
[x] bug report => see 'Providing a Reproducible Scenario'
[] feature request => do not use Github for feature requests, see 'Customers of AG Grid'
[x] support request => see 'Requesting Community Support'
```

**Requesting Community Support**

Hello when I update version 27.0.1, the floatingFilter not keep value when I call gridOptions.api.setRowData(data).
I already set
```
filterParams: { 
    newRowsAction: 'keep',
}
```
but it only work on version 26.2.1

**Providing a Reproducible Scenario**
A plunker link:
https://plnkr.co/edit/VNsfbizQZKYkoxLg

- Open it in plunker
- Typing random text on floatingFilter
- Click setRowData

**Current behavior**
The floatingFilter not keep value when call setRowData

**Expected behavior**
Keep value filter when call gridOptions.api.setRowData(data)

**Please tell us about your environment:**
Window 10

* **AG Grid version:** 27.0.1

* **Browser:** Chrome 98.0.4758.102

* **Language:** Javascript

**Comment by makinggoodsoftware** [comment]
Hi,

Your ticket has been flagged as in_zendesk, it means that we recognize this as an issue worth investigating so we have moved it to our official support channel for customers (zendesk)

We will carry on there with the investigation and we will update you as soon as possible.

Thanks

**Comment by icedjello** [comment]
Hi, 

Thank you for submitting this issue with an example. 

This issue has been resolved in v27.1.0. 

Please see the example below showing this: 

https://plnkr.co/edit/3JesL8lGMllo5Xcs 

I hope this helps,

**Comment by nguyenhuylinhdev** [comment]
Thank you so much, this worked!
